graphql/schema.graphql

```diff
@@ -142,20 +142,37 @@ type AnnotatedForm {
 	Unique identifier of the containing document
 	"""
 	documentId: UUID!
+	"""
+	Unique identifier of this form
+	"""
+	id: UUID!
 }
 
 """
 A single word in an annotated document that can be edited.
+All fields except id are optional.
 """
 input AnnotatedFormUpdate {
 	"""
 	Unique identifier of the form
 	"""
 	id: UUID!
 	"""
-	Original source text that can be either undefined or null
+	The document id where the given form lives
+	"""
+	docId: UUID!
```

Comment:
Makes sense - I had this because I thought it would be easier to send the document id back from the front-end to the back-end so calculating it in the back-end wouldn't be necessary, but that might make it more cluttered.